src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -78,28 +102,8 @@ const apolloMocks: MockedResponse[] = [
       data: { myProfile: fakePerson({ organisations: fakeOrganisations() }) },
     },
   },
-  {
-    request: {
-      query: EventDocument,
-      variables: {
-        id: eventMock.id,
-        include: ['location'],
-      },
-    },
-    result: {
-      data: {
-        event: {
-          ...eventMock,
-          pEvent: fakePEvent({
-            occurrences: fakeOccurrences(
-              fakeOccurrenceOverrides.length,
-              fakeOccurrenceOverrides
-            ),
-          }),
-        },
-      },
-    },
-  },
+  eventMockedResponse,
+  eventMockedResponse,
```

Comment:
Is there a reason to have this mock response twice?

Reply:
Yeah. It needs the second one for the event refetch request when saving occurrence. I will add a comment